### PR TITLE
Fix menus issues and remove most errors

### PR DESCRIPTION
**Pull request by Zach777**
Notably there is one error that I have no idea how to fix without breaking things.
 It is the only error that pops up and it has something to do with an  Expression.

### CODE REVIEW

File: _tests/behaviors/Definitions.gd [hunk 1]
```diff
@@ -4,7 +4,7 @@ class_name NpcDefinitions
 #Class ID will be assigned from 27 and on, in the order classes are declared
 func _init():
```
Comment (yaelatletl) [inline _tests/behaviors/Definitions.gd:5]: This function can be safely deleted, it's left over testing code, same goes for misc()

Comment (Zach777) [reply _tests/behaviors/Definitions.gd:5]: @yaelatletl Sorry for late response. I would have responded sooner if I had been alerted of this message.
 Is it alright if I delete Definitions.gd entirely?

**Comment by Reousa** [comment]
Is this PR still relevant?

**Comment by Zach777** [comment]
Not sure. @yaelatletl Do you still need the npc editor menu fixes here?

**Pull request closed without merging**